frontend/src/metabase/lib/data_grid.js

```diff
@@ -87,8 +87,8 @@ function createRowSectionGetter({
 }) {
   const formatValues = values =>
     values === undefined
-      ? new Array(valueFormatters.length).fill(null)
-      : values.map((v, i) => valueFormatters[i](v));
+      ? new Array(valueFormatters.length).fill({ value: null })
```

Comment:
Nitpick (we probably ought to standardize on this?): no need for `new`.

Reply:
Oh neat I didn't realize that worked! I'm good either way. Without `new` is more concise, but with `new` is consistent with `Set` 🤷 . I updated it here.